atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownTableManipulationTest.java

```diff
@@ -16,103 +16,73 @@
-        // This documents and verifies the current behaviour, dropping the table in spite of the exception
-        // Seems to be inconsistent with the API
-        assertThat(OneNodeDownTestSuite.kvs.getAllTableNames()).doesNotContain(OneNodeDownTestSuite.TEST_TABLE_TO_DROP);
-    }
+    public void canDropTable() {
+        getTestKvs().dropTable(TABLE_TO_DROP);
 
-    @Test
-    public void dropTablesThrows() {
-        assertThat(OneNodeDownTestSuite.kvs.getAllTableNames()).contains(OneNodeDownTestSuite.TEST_TABLE_TO_DROP_2);
-        assertThatThrownBy(() -> OneNodeDownTestSuite.kvs.dropTables(
-                ImmutableSet.of(OneNodeDownTestSuite.TEST_TABLE_TO_DROP_2)))
-                .isExactlyInstanceOf(AtlasDbDependencyException.class)
-                .hasCauseInstanceOf(UncheckedExecutionException.class);
-        // This documents and verifies the current behaviour, dropping the table in spite of the exception
-        // Seems to be inconsistent with the API
-        assertThat(OneNodeDownTestSuite.kvs.getAllTableNames())
-                .doesNotContain(OneNodeDownTestSuite.TEST_TABLE_TO_DROP_2);
-    }
-
-    @Test
-    public void canCompactInternally() {
-        OneNodeDownTestSuite.kvs.compactInternally(OneNodeDownTestSuite.TEST_TABLE);
+        assertThat(getTestKvs().getAllTableNames()).doesNotContain(TABLE_TO_DROP);
+        assertKvsReturnsEmptyMetadata(TABLE_TO_DROP);
+        assertCassandraSchemaChanged();
     }
 
     @Test
-    public void canCleanUpSchemaMutationLockTablesState() throws Exception {
-        ImmutableCassandraKeyValueServiceConfig config = OneNodeDownTestSuite.CONFIG;
-        CassandraClientPool clientPool = OneNodeDownTestSuite.kvs.getClientPool();
-        SchemaMutationLockTables lockTables = new SchemaMutationLockTables(clientPool, config);
-        TracingQueryRunner queryRunner = new TracingQueryRunner(LoggerFactory.getLogger(TracingQueryRunner.class),
-                new TracingPrefsConfig());
-        CassandraSchemaLockCleaner cleaner = CassandraSchemaLockCleaner.create(config, clientPool, lockTables,
-                queryRunner);
+    public void canDropTables() {
+        getTestKvs().dropTables(ImmutableSet.of(TABLE_TO_DROP_2));
 
-        cleaner.cleanLocksState();
+        assertThat(getTestKvs().getAllTableNames()).doesNotContain(TABLE_TO_DROP_2);
+        assertKvsReturnsEmptyMetadata(TABLE_TO_DROP_2);
+        assertCassandraSchemaChanged();
     }
 
     @Test
     public void truncateTableThrows() {
-        assertThatThrownBy(() -> OneNodeDownTestSuite.kvs.truncateTable(OneNodeDownTestSuite.TEST_TABLE))
-                .isExactlyInstanceOf(InsufficientConsistencyException.class)
-                .hasMessage("Truncating tables requires all Cassandra nodes to be up and available.");
+        assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(() ->
+                getTestKvs().truncateTable(TEST_TABLE));
```

Comment:
Looks like we need all to truncate, but _not_ all to drop, which seems inconsistent. @tpetracca are we aware of any reasons to hold back on this?